tests/security_features_test.go

```diff
@@ -40,48 +34,11 @@ import (
 var _ = Describe("SecurityFeatures", func() {
 	tests.FlagParse()
 
-	var originalKubeVirtConfig *k8sv1.ConfigMap
 	virtClient, err := kubecli.GetKubevirtClient()
 	tests.PanicOnError(err)
 
-	tests.BeforeAll(func() {
-		originalKubeVirtConfig, err = virtClient.CoreV1().ConfigMaps(tests.KubeVirtInstallNamespace).Get("kubevirt-config", metav1.GetOptions{})
-		if err != nil && !errors.IsNotFound(err) {
-			Expect(err).ToNot(HaveOccurred())
-		}
-
-		if errors.IsNotFound(err) {
-			// create an empty kubevirt-config configmap if none exists.
-			cfgMap := &k8sv1.ConfigMap{
-				ObjectMeta: metav1.ObjectMeta{Name: "kubevirt-config"},
-				Data: map[string]string{
-					"feature-gates": "",
-				},
-			}
-
-			originalKubeVirtConfig, err = virtClient.CoreV1().ConfigMaps(tests.KubeVirtInstallNamespace).Create(cfgMap)
-			Expect(err).ToNot(HaveOccurred())
-		}
-	})
-
-	AfterEach(func() {
-		curKubeVirtConfig, err := virtClient.CoreV1().ConfigMaps(tests.KubeVirtInstallNamespace).Get("kubevirt-config", metav1.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
-
-		// if revision changed, revert ConfigMap
-		if curKubeVirtConfig.ResourceVersion != originalKubeVirtConfig.ResourceVersion {
-			// Add Spec Patch
-			newData, err := json.Marshal(originalKubeVirtConfig.Data)
-			Expect(err).ToNot(HaveOccurred())
-			data := fmt.Sprintf(`[{ "op": "replace", "path": "/data", "value": %s }]`, string(newData))
-
-			originalKubeVirtConfig, err = virtClient.CoreV1().ConfigMaps(tests.KubeVirtInstallNamespace).Patch("kubevirt-config", types.JSONPatchType, []byte(data))
-			Expect(err).ToNot(HaveOccurred())
-
-			// Allow time for virt-controller's ConfigMap cache to sync
-			time.Sleep(3 * time.Second)
-		}
-
+	BeforeEach(func() {
+		tests.BeforeTestCleanup()
```

Comment:
Wondering if this would create an empty kubevirt-config cfgmap if non exists? At-least in downstream 
I know of that it does exist always. Not sure if it is the same in upstream.

Reply:
Anyways all tests in all the lanes seem to have passed.

Reply:
We always create a config-map before we start the tests. Should be good.